### PR TITLE
Setting Translatable = false

### LINKED ISSUE #756: BUG: aapt d badging fails for translated apps.
### Bug Report
aapt dump badging fails for recommended metadata setup on translatable apps

##### Reproduction Steps
1. [Step 1]
2. [Step 2]
3. [Step 3]

##### Expected Behavior
[Some expected behavior]

##### Observed Behavior
AndroidManifest.xml:635: error: ERROR getting 'android:name' attribute for meta-data tag in service 'com.accuweather.ford.SdlRouterService': attribute is not a string value

##### OS & Version Information
* SDL Android Version:4.50

##### Test Case, Sample Code, and / or Example App
In manifest:
       <service
            android:name="com.accuweather.ford.SdlRouterService"
            android:exported="true"
            android:process="com.smartdevicelink.router">

            <intent-filter>
                <action android:name="com.smartdevicelink.router.service" />
            </intent-filter>
            <meta-data
                android:name="@string/sdl_router_service_version_name"
                android:value="@integer/sdl_router_service_version_value" />

        </service>

This passes when replaced with hardcoded values.
 see https://smartdevicelink.com/en/guides/android/migrating-to-newer-sdl-android-versions/updating-to-45/

### PR DESCRIPTION
**Pull request by MongoTheGeek**
Fixes #756 

This PR is **ready** for review.

### Risk
This PR makes **no** API changes.

### Testing Plan
Build translatable app.
run adapt d badging against

### Summary
set translatable false to non translated string

##### Bug Fixes
Marked version texts as non-translatable.

### CLA
- [X ] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: sdl_android/src/main/res/values/sdl.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="sdl_router_service_version_name">sdl_router_version</string>
-    <integer name="sdl_router_service_version_value">5</integer>
+    <string name="sdl_router_service_version_name" translatable="false">sdl_router_version</string>
+    <integer name="sdl_router_service_version_value" translatable="false">5</integer>
```
Comment (bilal-alsharifi) [inline sdl_android/src/main/res/values/sdl.xml]: I don't think "translatable" applies to integers.

**Comment by bilal-alsharifi** [comment]
Thanks @MongoTheGeek, the PR looks good. Setting translateable to false is necessary for strings. But can you please remove the "translatebble=false" for the integer value? And also please target develop branch instead of master. So I can merge the PR.

File: sdl_android/src/main/res/values/sdl.xml (hunk 1 above)
Comment (brettywhite) [reply sdl_android/src/main/res/values/sdl.xml]: Hi @MongoTheGeek Are you able to make the changes?

Comment (MongoTheGeek) [reply sdl_android/src/main/res/values/sdl.xml]: Sorry. I have been back packing across Spain. I can make the changes in a couple of days.

**Pull request closed without merging**